cmake/psi4OptionsTools.cmake

```diff
@@ -62,6 +62,12 @@ macro(psi4_add_module binlib libname sources)
     PRIVATE
       tgt::lapack
     )
+  if(MSVC)
```

Comment:
Was there a reason my MSVC had a special case here? I know it’s been a while since it was added. Just curious.

Reply:
I tested again and win fails without.